### PR TITLE
backport: cmake prereqs 1

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
## Issue being fixed or feature implemented
First batch of PRs which lead towards cmake, focusing on having nothing or very little done out of order / with significant conflicts

## What was done?
batch of 11 backports which lead towards cmake

## How Has This Been Tested?
Built locally, seems to work, let's see CI

## Checklist:
  _Go over all the following points, and put an `x` in all the boxes that apply._
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation
- [x] I have assigned this pull request to a milestone _(for repository code-owners and collaborators only)_

### CODE REVIEW

**Review by UdjinM6 — APPROVED** [review]
utACK 801c4fc24a1493c211022f2dde48892c6fae3f3f

File: depends/packages/qrencode.mk [hunk 1]
```diff
@@ -1,15 +1,16 @@
 package=qrencode
-$(package)_version=3.4.4
+$(package)_version=4.1.1
```
Comment (knst) [inline depends/packages/qrencode.mk:2]: nit 27312: missing update in `doc/dependencies.md`
consider backporting `bitcoin/bitcoin#27441`

File: depends/packages/zeromq.mk [hunk 2]
```diff
@@ -1,9 +1,9 @@
 package=zeromq
-$(package)_version=4.3.4
+$(package)_version=4.3.5
```
Comment (knst) [inline depends/packages/zeromq.mk:2]: nit-28627: doc/dependencies.md is not updated but there's no backport with these changes, I guess someday

**Review by knst — APPROVED** [review]
utACK  801c4fc24a1493c211022f2dde48892c6fae3f3f

**Pull request merged**